### PR TITLE
webalizer 2.23-08

### PR DESCRIPTION
**Pull request by dunn**

### CODE REVIEW

File: Library/Formula/webalizer.rb [hunk 1]
```diff
@@ -1,26 +1,31 @@
-require 'formula'
-
 class Webalizer < Formula
   homepage "http://www.mrunix.net/webalizer/"
```
Comment (vszakats) [inline Library/Formula/webalizer.rb]: Maybe better to follow the permanent redirect to this address?:
http://www.webalizer.org/

File: Library/Formula/webalizer.rb [hunk 2]
```diff
@@ -1,26 +1,31 @@
...
-    system "make install"
+                          "--prefix=#{prefix}",
+                          "--mandir=#{man}"
+    system "make", "install"
+  end
+
+  test do
+    (testpath/"test.log").write "127.0.0.1 user-identifier homebrew [10/Oct/2000:13:55:36 -0700] \"GET /beer.gif HTTP/1.0\" 200 2326"
+    system "#{bin}/webalizer", "-c", etc/"webalizer.conf.sample", testpath/"test.log"
+    File.exist? "usage.png"
```
Comment (DomT4) [inline Library/Formula/webalizer.rb]: Should be `assert File.exist?`

Comment (dunn) [inline Library/Formula/webalizer.rb]: OK, but without an assertion message it's not much more informative is it?

Comment (tdsmith) [inline Library/Formula/webalizer.rb]: Yeah, this is fine; if the `test` block returns false, the test fails.

**Comment by xu-cheng** [comment]
Thanks @dunn

**Pull request closed without merging**